lego/apps/email/notifications.py

```diff
@@ -0,0 +1,120 @@
+from datetime import timedelta
+
+from django.utils import timezone
+
+from lego.apps.events.models import Event
+from lego.apps.joblistings.models import Joblisting
+from lego.apps.notifications.constants import WEEKLY_MAIL
+from lego.apps.notifications.notification import Notification
+from lego.apps.permissions.constants import VIEW
+from lego.apps.tags.models import Tag
+
+JOB_TYPE_MAPPINGS = {
+    "summer_job": "Sommerjobb",
+    "part_time": "Deltid",
+    "full_time": "Fulltid",
+    "master_thesis": "Masteroppgave",
+    "other": "Annet",
+}
+
+EVENT_TYPE_MAPPINGS = {
+    "company_presentation": "Bedriftspresentasjon",
+    "lunch_presentation": "Lunchpresentasjon",
+    "alternative_presentation": "Alternativ presentasjon",
+    "course": "Kurs",
+    "breakfast_talk": "Frokostforedrag",
+    "kid_event": "KID-arrangement",
+    "party": "Fest",
+    "social": "Sosialt",
+    "other": "Annet",
+    "event": "Arrangement",
+}
```

Comment:
I would prefer if you'd use the mappings in `lego.apps.companies.constants.py`. We don't have a direct mapping, so you'll have to do some trickery with what we've got though. Or, move this to the file, if it turns out very ugly. The same kinda goes for `JOB_TYPE_MAPPINGS` above, please move it.

Reply:
Yes, I will move it. Translated events is used more for company interest than actual events, so I don't think it can be used here. I think we will benefit from having a separate dictionary or having one that is merged with this